docs/api/covidcast-signals/google-symptoms.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Google Symptoms
+parent: Data Sources and Signals
+grand_parent: COVIDcast API
+---
+
+# Google Symptoms
+{: .no_toc}
+
+* **Source name:** `google-symptoms`
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** county, MSA, HRR, state (see [geography coding docs](../covidcast_geography.md))
+
+This data source is based on [COVID-19 Search Trends symptoms dataset](https://github.com/google-research/open-covid-19-data/tree/master/data/exports/search_trends_symptoms_dataset). Using this search data, we estimate the volume of searches mapped to symptoms related 
+to COVID-19 such as _anosmia_(lack of smell) and _ageusia_(lack of taste). The resulting daily dataset for each region 
+showing the relative frequency of searches for each symptom.  This signal is measured 
+in arbitrary units that are normalized for population and for the most popular symptom 
+search term within a geographic region. Thus, values are not comparable between 
+geographic regions. Larger numbers represent higher numbers of symptom-related 
+searches.
+
+| Signal | Description |
+| --- | --- |
+| `anosmia_raw_search` |  Google search volume for anosmia-related searches, in arbitrary units that are normalized for population |
+| `anosmia_smoothed_search` | Google search volume for anosmia-related searches, in arbitrary units that are normalized for population, smoothed by 7-day average |
+| `ageusia_raw_search` | Google search volume for ageusia-related searches, in arbitrary units that are normalized for population |
+| `ageusia_smoothed_search` |  Google search volume for ageusia-related searches, in arbitrary units that are normalized for population, smoothed by 7-day average |
+| `combined_symptoms_raw_search` | wait for Roni's decision |
+| `combined_symptoms_smoothed_search` | wait for Roni's decision |
+
+
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+## Estimation (for combined_symptoms)
+wait for Roni's decision
+
+## Limitation 
+When daily volume in a region does not meet quality or privacy thresholds, set by Google, the value
+is assigned a NaN and will not be reported. Since Google uses differential privacy, there is artificial 
```

Comment:
```suggestion
When daily volume in a region does not meet quality or privacy thresholds, set by Google, no value
will be reported. Since Google uses differential privacy, there is artificial 
```